test/gie/peirce_q.gie

```diff
@@ -5,14 +5,950 @@
 ------------------------------------------------------------
 
 ------------------------------------------------------------
-operation  +proj=peirce_q +R=6370997
+operation  +proj=peirce_q +R=6370997 +type=diamond
 tolerance  1 mm
 ------------------------------------------------------------
 accept    -179.6126302052  -90.2440064745
 expect    failure errno coord_transfm_invalid_coord
 
 accept    -169.8486692749  -89.5742075028
-expect    failure errno coord_transfm_outside_projection_domain
+expect         -8344.698       23578003.421
```

Comment:
I'm not sure we need that many test points, but we'd need tests for the other +type and new parameters

Reply:
have reduced number of test points. probably could be rationalised further, though for now have carried over a selection from the points already included